scripts/get_message_templates.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+path="$1"
+function getMessageTemplates() {
+  grep -R "\\.withMessageTemplate" --exclude="*Test*" --include="*.java" "$path" | awk -F'"' '{sub(/[ \t]+$/, "", $2); sub(/[ \t]+$/, "", $3); print $2, $3, $4}' | sort -g | sed 's/,/=/' > temp.properties
```

Comment:
The "exclude" and "include" should be switched in order to really exclude the test java files.